app/src/components/governance/ProposalDescription.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="proposal-description">
+    <section>
+      <h4>Description</h4>
+      <p class="description">
+        {{ description }}
+      </p>
+    </section>
+    <aside class="supporting-links">
+      <h4>Supporting Links</h4>
+      <ul>
+        <li v-for="link in supportingLinks" :key="link.url">
+          <a href="link.url">{{ link.title }}</a>
+        </li>
+      </ul>
+    </aside>
+  </div>
+</template>
+
+<script>
+export default {
+  name: `proposal-description`,
+  data: () => ({
+    description: `Stargate is our name for the process of ensuring that the widely integrated public network known as the Cosmos Hub is able to execute the cosmoshub-3 -> cosmoshub-4 upgrade with the minimum disruption to its existing ecosystem. This upgrade will also realize the Internet of Blockchains vision from the Cosmos whitepaper.
```

Comment:
can we use markdown in our db? if so, how will the data be returned to the FE? in one blob?

Reply:
well, if it is a string/text, it will just store it as it comes. Should get returned the same as it entered, ideally. Just check

Reply:
how would you display markdown? problem is, that you would need to render this as HTML which is dangerous. we discussed this about 1.5 years ago

Reply:
is it dangerous if it's coming from our db / not user generated?